### PR TITLE
[Qt module] Activate Qt tests on OSX CI

### PR DESCRIPTION
**Pull request by jeandet**
Ugly hack..., But looking the code, this an easy way to just activate qt test in OSX.

### CODE REVIEW

**Comment by jpakkane** [comment]
But does not seem to work ...

**Comment by jeandet** [comment]
Yep qt4 isn't installed, need some local tests before :)

**Comment by jpakkane** [comment]
Seems to be still broken. And :crossed_swords: conflicts.

**Comment by jeandet** [comment]
@jpakkane you can close few of my long standing PRs, I'm really busy now and won't touch meson code for the next few weeks. So, if you want to do some cleaning inside PRs do not hesitate :)

File: .travis.yml [hunk 1]
```diff
@@ -49,4 +50,4 @@ script:
         withgit \
           /bin/sh -c "cd /root && mkdir -p tools; wget -c http://nirbheek.in/files/binaries/ninja/linux-amd64/ninja -O /root/tools/ninja; chmod +x /root/tools/ninja; CC=$CC CXX=$CXX OBJC=$CC OBJCXX=$CXX PATH=/root/tools:$PATH MESON_FIXED_NINJA=1 ./run_tests.py -- $MESON_ARGS && chmod -R a+rwX .coverage"
     fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then SDKROOT=$(xcodebuild -version -sdk macosx Path) OBJC=$CC OBJCXX=$CXX PATH=$HOME/tools:$PATH MESON_FIXED_NINJA=1 ./run_tests.py --backend=ninja -- $MESON_ARGS ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then alias qmake-qt4=/usr/local/Cellar/qt@4/4.8.7_3/bin/qmake; SDKROOT=$(xcodebuild -version -sdk macosx Path) OBJC=$CC OBJCXX=$CXX PATH=$HOME/tools:/usr/local/Cellar/qt/5.9.2/bin:$PATH MESON_FIXED_NINJA=1 ./run_tests.py --backend=ninja -- $MESON_ARGS ; fi
```
Comment (jon-turney) [inline .travis.yml]: The path added to PATH probably needs changing from /usr/local/Cellar/qt/5.9.2/bin to /usr/local/opt/qt/bin to ensure this doesn't break when the version of Qt in homebrew changes.

**Comment by nirbheek** [comment]
@jeandet did you get time to look at this again?

**Comment by jeandet** [comment]
@nirbheek , looking at this https://travis-ci.org/mesonbuild/meson/jobs/387445454, it seems Qt tests are already running on OSX.

**Comment by nirbheek** [comment]
> it seems Qt tests are already running on OSX.

`Succeeded test: test cases/frameworks/4 qt`

Looks like, it closing, thanks!

**Pull request closed without merging**